e2e-tests/spring-rest-openapi-v2/src/test/java/org/evomaster/e2etests/spring/examples/adaptivehypermutation/AHypermutationAWHTest.java

```diff
@@ -111,6 +111,8 @@ public void testRunMIOAWH() throws Throwable {
                 budget,
                 true,
                 (args) -> {
+                    args.add("--testSuiteSplitType");
+                    args.add("NONE");
 
```

Comment:
adding this on every single E2E is not viable, nor scalable.
should rather be default in `WsTestBase`, with option to use other modes if needed